web/html/src/manager/maintenance/calendar/web-calendar.test.tsx

```diff
@@ -249,7 +260,9 @@ describe("Web calendar", () => {
                         date={initialDate}
     />)
 
-    screen.getByText("July 2021");
+    await waitFor(() => {
```

Comment:
Can this perhaps be merged with the `waitFor` call below? Not sure what the logic is here, just something that caught my eye.

Reply:
Yes, it can. That was an oversight from me side. Thanks for pointing it out :)